packages/delayedInbox-l2msg/README.md

```diff
@@ -2,7 +2,9 @@
 (Noted this can only be done in nitro stack, not in arbitrum classic, so l2 rpc please use nitro test rpc)
 delayedInbox-l2msg is a simple sample example that allows you to send a l2 msg without using sequencer way, this can be used when sequencer censor your tx or when sequencer downs.
```

Comment:
delayedInbox-l2msg is a simple sample example that allows you to send a l2 msg without using sequencer way, this can be used when sequencer censors your tx or when sequencer is down.